lib/index.js

```diff
@@ -12,7 +12,12 @@ var ParserInline = require('./parser_inline');
 var LinkifyIt    = require('linkify-it');
 var mdurl        = require('mdurl');
 var punycode     = require('punycode');
+var trimRight    = require('trim-right');
 
+if (!String.prototype.trimRight) {
```

Comment:
I mean, that a lot of my comments can look annoying in your eyes :). But that's not intentional and i promis that tables fix will land soon.